packages/realm-react/package.json

```diff
@@ -16,7 +16,7 @@
   },
   "wireit": {
     "bundle": {
-      "command": "rollup --config",
+      "command": "rollup --config && tsc -p tsconfig.types.json",
```

Comment:
Should this be a separate `types` command? Since technically this isn't bundling the types now.

Reply:
As our ci assumes a single command for preparing packages for release, I would be hesitant to put this in a separate command.
Otherwise, we can make a follow up PR which adds a `types` command to every package (or simply `build`, which contains `bundle` and `types`) and update our workflows to call this as well.

Reply:
I put this in our next roundtable.  I think you bring up a good point.

My suggestion is to create a script in each of our mono-repo packages called `prepack`, which can include whatever scripts it needs to prepare for being consumed by another project.  This could include `bundle`, `generateTypes`, `build`, etc.
Then the CI workflows would be updated to run this command instead of `bundle`.

Reply:
It was the idea that these inter-package task dependencies should be configured via wireit. It's much more capable as it'll cache outputs and potentially execute tasks in parallel.

Reply:
I don't disagree, I just want to unify on a script that specifically deals with preparing for release.